CLI_QUICKSTART.markdown

```diff
@@ -344,4 +344,18 @@ with a pkill command:
 $ pkill vc-server
 $ pkill vc-pas
 ```
+##  IP addresses and ports used by various Veracruz components
+
+|    IP      | Port |                              Usage                                                         | Location(s) |
+|------------|------|--------------------------------------------------------------------------------------------|-------------|
+| 127.0.0.1  | 6000 | Port used by Veracruz Server and Runtime Manager to communicate over a TCP socket          | `veracruz-server/src/veracruz_server_linux.rs` |
+| 127.0.0.1  | 3016 | Port used in test for proxy attestion service                                              | `proxy-attestation-server/src/test.rs` |
+| 127.0.0.1  | 3010 | Port used for proxy attestation server in MCU test, veracruz-server-test and veracruz-test | `veracruz-mcu-client/run_mcu_demo_server.sh` <br><br> `veracruz-mcu-client/run_mcu_test_server.sh` <br><br> `veracruz-mcu-client/MCU_CLIENT_INSTRUCTIONS.markdown` <br><br> `workspaces/shared.mk` <br><br> `docker/Makefile`|
```

Comment:
Can you capitalise Proxy Attestation Server, here, and throughout the rest of the file?